src/precompile_utils.c

```diff
@@ -188,7 +188,7 @@ static int precompile_enq_specialization_(jl_method_instance_t *mi, void *closur
                 (jl_ir_inlining_cost((jl_array_t*)inferred) == UINT16_MAX)) {
                 do_compile = 1;
             }
-            else if (jl_atomic_load_relaxed(&codeinst->invoke) != NULL || jl_atomic_load_relaxed(&codeinst->precompile)) {
+            else if (jl_atomic_load_relaxed(&codeinst->precompile)) {
```

Comment:
Maybe instead we can check that `invoke` is not set to `jl_fptr_interpret_call`?

Reply:
Yes, that would work for this case, but I don't really understand the purpose and semantics of the `->precompile` field then if we just basically ignore it here.

Reply:
@timholy wrote this particular section of code.